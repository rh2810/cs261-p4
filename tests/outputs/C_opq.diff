--- outputs/C_opq.txt	2017-11-09 17:26:42.572822840 -0500
+++ expected/C_opq.txt	2017-11-03 11:51:30.000000000 -0400
@@ -0,0 +1,12 @@
+Beginning execution at 0x0100
+Y86 CPU state:
+  %rip: 0000000000000000   flags: Z0 S0 O0     HLT
+  %rax: 0000000000000008    %rcx: 0000000000000001
+  %rdx: 0000000000000006    %rbx: 0000000000000002
+  %rsp: 0000000000000000    %rbp: 0000000000000000
+  %rsi: 0000000000000000    %rdi: 0000000000000003
+   %r8: 0000000000000000     %r9: 0000000000000000
+  %r10: 0000000000000000    %r11: 0000000000000000
+  %r12: 0000000000000000    %r13: 0000000000000000
+  %r14: 0000000000000000
+Total execution count: 10
