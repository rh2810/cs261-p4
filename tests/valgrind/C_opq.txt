==18440== Memcheck, a memory error detector
==18440== Copyright (C) 2002-2015, and GNU GPL'd, by Julian Seward et al.
==18440== Using Valgrind-3.11.0 and LibVEX; rerun with -h for copyright info
==18440== Command: ../y86 -e inputs/opq.o
==18440== 
==18440== 
==18440== HEAP SUMMARY:
==18440==     in use at exit: 0 bytes in 0 blocks
==18440==   total heap usage: 0 allocs, 0 frees, 0 bytes allocated
==18440== 
==18440== All heap blocks were freed -- no leaks are possible
==18440== 
==18440== For counts of detected and suppressed errors, rerun with: -v
==18440== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
