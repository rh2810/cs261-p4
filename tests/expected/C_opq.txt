Beginning execution at 0x0100
Y86 CPU state:
  %rip: 0000000000000000   flags: Z0 S0 O0     HLT
  %rax: 0000000000000008    %rcx: 0000000000000001
  %rdx: 0000000000000006    %rbx: 0000000000000002
  %rsp: 0000000000000000    %rbp: 0000000000000000
  %rsi: 0000000000000000    %rdi: 0000000000000003
   %r8: 0000000000000000     %r9: 0000000000000000
  %r10: 0000000000000000    %r11: 0000000000000000
  %r12: 0000000000000000    %r13: 0000000000000000
  %r14: 0000000000000000
Total execution count: 10
